PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/BubbledGraphTest.java

```diff
@@ -0,0 +1,21 @@
+package nl.tudelft.pl2016gr2.model;
+
+public class BubbledGraphTest extends GraphTest {
+  
+  /**
+   * Setup, creates a new graph.
+   */
```

Comment:
If you write comments in the parent class you may leave out the comments for overwritten methods in the child class (if no additional description is needed). The child class will simply inherit the comments from the parent class.